server/asset/ltc/ltc.go

```diff
@@ -10,7 +10,30 @@ import (
 	dexbtc "decred.org/dcrdex/dex/btc"
 	"decred.org/dcrdex/server/asset"
 	"decred.org/dcrdex/server/asset/btc"
-	"github.com/ltcsuite/ltcd/chaincfg"
+)
+
+var (
+	// MainNetParams are the clone parameters for mainnet.
+	MainNetParams = &dexbtc.CloneParams{
+		PubKeyHashAddrID: 0x30,
+		ScriptHashAddrID: 0x32,
+		Bech32HRPSegwit:  "ltc",
+		CoinbaseMaturity: 100,
```

Comment:
Could you add the Net to these. The client needs it when calling chaincfg.Register which is needed to properly decode bech32 addresses here: https://github.com/btcsuite/btcutil/blob/b2bf7f89d674a3702182b7e15f52807896051af3/address.go#L145

Reply:
Can do.